theseus/labs/lie/functional/lie_group.py

```diff
@@ -169,6 +169,16 @@ def __call__(
         pass
 
 
+class _IdentityFnType(Protocol):
```

Comment:
Offline discussion: won't add eye for now since it is confusing in the context of groups (especially when storage is quaternion as pointed out by Riku).